src/lib/tabs/tab-body.ts

```diff
@@ -146,7 +146,8 @@ export class MdTabBody implements OnInit, AfterViewChecked {
    */
   ngAfterViewChecked() {
     if (this._isCenterPosition(this._position) && !this._portalHost.hasAttached()) {
-      this._portalHost.attach(this._content);
+      // Nested templates via mdTabContent templates causes expression change error
+      Promise.resolve().then(() => this._portalHost.attach(this._content));
```

Comment:
Can we attach it in `ngDoCheck` instead? It would be better to avoid the extra change detection pass.

Reply:
Note: After this change I had to update the unit test that checked tab body contents.